### PR TITLE
NOISSUE - Add Rules service page to documentation

### PR DESCRIPTION
**Pull request by darkodraskovic**
Signed-off-by: Darko Draskovic <darko.draskovic@gmail.com>

### CODE REVIEW

File: docs/rules.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
+# Rules
+
+## About service
+
+## Start service
+
+
```
Comment (drasko) [inline docs/rules.md:41]: Kill those unnecessary empty lines

**Review by drasko — CHANGES_REQUESTED** [review]

File: docs/rules.md [hunk 2]
```diff
@@ -0,0 +1,435 @@
+# Rules
+
+## About service
+
+Mainflux Rules service is build around [EMQ X Kuiper](https://github.com/emqx/kuiper), a lightweight IoT rules engine. Mainflux Rules service consists of
```
Comment (manuio) [inline docs/rules.md:5]: Maybe you can use `:` after `of`?

File: docs/rules.md [hunk 3]
```diff
@@ -0,0 +1,435 @@
...
+Response
+
+```
+{
+  "email": "pensive_galileo@email.com",
+  "password": "12345678"
+}
+
+
+"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE2MjA0MjgxODAsImlhdCI6MTYyMDM5MjE4MCwiaXNzIjoibWFpbmZsdXguYXV0aCIsInN1YiI6InBlbnNpdmVfZ2FsaWxlb0BlbWFpbC5jb20iLCJpc3N1ZXJfaWQiOiI0NDlhMGE5YS04YjE5LTQwMTgtODA3My05YzZjMmM2ZDhjYzkiLCJ0eXBlIjowfQ.QFjP27WAAy8OUt6yQzjLjk8HVO9CMeKNQThwYYHcPIA"
```
Comment (manuio) [inline docs/rules.md:67]: There is a token in the response? If yes, why it's not a JSON?

**Comment by drasko** [comment]
@darkodraskovic please resolve these and let's move with this PR

**Comment by drasko** [comment]
Closing, as we moved away from Kuiper RE

**Pull request closed without merging**